src/templates/icon.overrides.liquid

```diff
@@ -1,3 +1,146 @@
 /*******************************
          Theme Overrides
 *******************************/
+
+/* Make duotone icons use the proper font */
+i.icon.duotone {
+  font-family: @duotoneFontName;
+}
+
+/* Position both icon layers */
+i.icon.duotone {
+  /* To position the secondary layer on top of the first layer */
+  position: relative;
+}
+
+i.icon.duotone::after {
+  /* Position secondary layer to the left, centered horizontally and aligned vertically to flex with different line heights */
+  position: absolute;
+  left: 0;
+  top: 50%;
+  transform: translateY(-50%);
+  text-align: center;
+}
+
+/* Set the default opacity levels and colors for each layer */
+i.icon.duotone::before {
+  color: @duotonePrimaryColor;
+  opacity: @duotonePrimaryOpacity;
+}
+
+i.icon.duotone::after {
+  color: @duotoneSecondaryColor;
+  opacity: @duotoneSecondaryOpacity;
+}
+
+
+/*
+ * Colors for duotone icons, in the form `primary-secondary`(e.g. `black-grey duotone icon`).
+ * Make sure to respect the `variation.variables` and `colors.less` file values when generating this
+ */
+
+/* Primary icon color */
+i.icon.duotone[class~="red-"]::before {
+  color: red;
```

Comment:
We should use the colors form the FUI color palette instead of the default browser constants which do not match the fui colors. 
So all color selectors have to be generated out of the colors.less file colors-map.
Let's do this inside the FUI code and move it out of this PR . Should i do this in the FUI Repo and would you like to do it?

Reply:
Ok help me out here :-)

Reply:
Does that mean that I should remove all color definitions from icon.overrides.liquid?

Reply:
> Does that mean that I should remove all color definitions from icon.overrides.liquid?

Yes